view/frontend/web/js/view/payment/method-renderer/adyen-hpp-method.js

```diff
@@ -193,10 +193,22 @@ define(
                             return 4;
                         }
                     };
-                    if (value.brandCode == "ideal") {
+                    result.isIssuerListAvailable = function () {
+                        if (typeof value.issuers !== 'undefined' && value.issuers.length > 0) {
+                            return true;
+                        }
+
+                        return false;
+                    };
+
+                    if (value.hasOwnProperty("issuers")) {
+                        if (value.issuers.length == 0) {
+                            return false;
+                        }
+
                         result.issuerIds = value.issuers;
                         result.issuerId = ko.observable(null);
-                    } else if (value.isPaymentMethodOpenInvoiceMethod) {
+                    }else if (value.isPaymentMethodOpenInvoiceMethod) {
```

Comment:
what formatter do you use? Think we can keep this space.

Reply:
Nothing extraordinary I usually follow how they style the js code:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/if...else
If we have a common setting I can set it up 👍